### PR TITLE
LPD-1894 Add sortable tooltip

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPD-1894

I followed the example made by Marko in this comment: https://liferay.atlassian.net/browse/LPD-1894?focusedCommentId=2519741

Let me know if there are any questions or comments about this.
Thank you!

### CODE REVIEW

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx [hunk 1]
```diff
@@ -580,11 +583,21 @@ const EditFDSFieldModalContent = ({
...
 						label={Liferay.Language.get('sortable')}
 						onChange={({target: {checked}}) =>
 							setFSDFieldSortable(checked)
 						}
 					/>
+
+					<span
+						className="label-icon lfr-portal-tooltip ml-2"
+						title={Liferay.Language.get(
+							'if-enabled-this-checkbox-defines-whether-the-field-can-be-sorted'
```
Comment (dsanz) [inline modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx:596]: I'd probably rewrite this key a bit, following the `if-checked` pattern, and stating that what is sorted is not the field itself but rather the data set items. How would this sound?:
`if-checked,-data-set-items-can-be-sorted-by-this-field`

Comment (fortunatomaldonado) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx:596]: Yeah that makes sense, thank you!
I'll wait for feedback from @pablo-agulla and @herrardo before making any changes.

**Review by markocikos — COMMENTED** [review]
@fortunatomaldonado apart from label that Dani mentioned, LGTM! 

A sidenote, you don't have to run `gradle buildLang`, it is enough just to add the change to Language.properties. This way, you may avoid some annoyance with conflicts.

**Comment by fortunatomaldonado** [comment]
ci:test:sf

File: modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx (hunk 1 above)
Comment (fortunatomaldonado) [reply modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx]: Made label changes.

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

**Comment by fortunatomaldonado** [comment]
Is this okay to forward now?
Thank you!

**Review by markocikos — APPROVED** [review]
@fortunatomaldonado LGTM! :sparkles:

**Comment by fortunatomaldonado** [comment]
ci:forward

**Pull request closed without merging**